registration/registration-services/src/test/java/io/mosip/registration/test/jobs/KeyPolicySyncJobTest.java

```diff
@@ -171,78 +177,15 @@ public void executejobNoSuchBeanDefinitionExceptionTest() {
 		keyPolicySyncJob.executeInternal(context);
+		ResponseDTO actual = keyPolicySyncJob.executeJob("triggerPoint", "jobId");
+		assertEquals(responseDTO, actual);
 	}
 
-	@Test(expected = RegBaseUncheckedException.class)
-	public void executejobNullPointerExceptionTest() {
-		ResponseDTO responseDTO = new ResponseDTO();
-		SuccessResponseDTO successResponseDTO = new SuccessResponseDTO();
-		responseDTO.setSuccessResponseDTO(successResponseDTO);
-		Mockito.when(context.getJobDetail()).thenThrow(NullPointerException.class);
-
-		keyPolicySyncJob.executeInternal(context);
-	}
-
-	@Test(expected = RegBaseUncheckedException.class)
-	public void executeChildJobsTest() throws JobExecutionException {
-		SyncJobDef syncJob = new SyncJobDef();
-		syncJob.setId("1");
-
-		Map<String, SyncJobDef> jobMap = new HashMap<>();
-
-		jobMap.put(syncJob.getId(), syncJob);
-
-		syncJob.setId("2");
-		syncJob.setParentSyncJobId("1");
-
-		jobMap.put("2", syncJob);
-
-		ResponseDTO responseDTO = new ResponseDTO();
-		SuccessResponseDTO successResponseDTO = new SuccessResponseDTO();
-		responseDTO.setSuccessResponseDTO(successResponseDTO);
-
-		Mockito.when(applicationContext.getBean(Mockito.anyString())).thenThrow(NoSuchBeanDefinitionException.class);
-
-		keyPolicySyncJob.executeParentJob("1");
-
-	}
-	
-	
-	@Test(expected = RuntimeException.class)
-	public void executejobRunTimeExceptionTest() throws ConnectionException, RegBaseCheckedException {
-		ResponseDTO responseDTO = new ResponseDTO();
-		ErrorResponseDTO errorResponseDTO = new ErrorResponseDTO();
-		List<ErrorResponseDTO> errorResponseDTOs = new ArrayList<>();
-		errorResponseDTOs.add(errorResponseDTO);
-		responseDTO.setErrorResponseDTOs(errorResponseDTOs);
-		SyncJobDef syncJob = new SyncJobDef();
-		syncJob.setId("1");
-
-		Map<String, SyncJobDef> jobMap = new HashMap<>();
-
-		jobMap.put(syncJob.getId(), syncJob);
-
-		syncJob.setId("2");
-		syncJob.setParentSyncJobId("1");
-
-		jobMap.put("2", syncJob);
-
-		Mockito.when(context.getJobDetail()).thenReturn(jobDetail);
-		Mockito.when(jobDetail.getJobDataMap()).thenReturn(jobDataMap);
-		Mockito.when(jobDataMap.get(Mockito.any())).thenReturn(applicationContext);
-		Mockito.when(applicationContext.getBean(SyncManager.class)).thenReturn(syncManager);
-		Mockito.when(applicationContext.getBean(JobManager.class)).thenReturn(jobManager);
-		Mockito.when(applicationContext.getBean(PolicySyncService.class)).thenReturn(policySyncService);
-
-		Mockito.when(jobManager.getJobId(Mockito.any(JobExecutionContext.class))).thenReturn("1");
-
-		Mockito.when(applicationContext.getBean(Mockito.anyString())).thenReturn(keyPolicySyncJob);
-
-		Mockito.when(policySyncService.fetchPolicy()).thenThrow(RuntimeException.class);
-
-		keyPolicySyncJob.executeInternal(context);
-
+	@Test(expected = RegBaseCheckedException.class)
+	public void testExecuteJobThrowsCheckedException() throws RegBaseCheckedException {
```

Comment:
We should start following the test case naming convention  @HimajaDhanyamraju2 
<methodNameBeingTested>_<testCase>_<testResult>
Example: executeJob_allValidInput_thenPass

Reply:
Created a separate task to rename the test cases in regclient to follow up. Will take it up as soon as the prioritized tasks are all done.
https://mosip.atlassian.net/browse/MOSIP-31278